src/setup/SetupPanel.ts

```diff
@@ -325,6 +331,34 @@ export class SetupPanel {
         case "exploreComponents":
           await commands.executeCommand("esp.component-manager.ui.show");
           break;
+        case "canAccessFile":
+          if (message.pathIdfPy) {
+            const fileExists = await canAccessFile(message.pathIdfPy);
```

Comment:
I think `message.pathIdfPy` can be generated here instead of generating it in the Vue UI.